.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
We should remind people to break up massive PRs into smaller chunks whenever possible. Definitely do not want to review massive PRs. 
So we could say something like `We prefer small, well tested pull requests. Please ensure your pull requests are self-contained, and commits are squashed`

Reply:
updated